python/pytest.ini

```diff
@@ -1,4 +1,4 @@
 [pytest]
 markers =
     smoke_test: mark a test as a build verification testing.
-addopts = -k "not test_redis_modules.py"
+addopts = --ignore "tests_redis_modules"
```

Comment:
Since it's already under tests folder, please rename the folder name to "redis_modules"

Reply:
What about changing the GitHub action that tests modules accordingly?

Reply:
no need to change a thing, the GitHub action ignores `addopts`

Reply:
( `pytest --asyncio-mode=auto --override-ini=addopts= --load-module=$GITHUB_WORKSPACE/redisearch.so --load-module=$GITHUB_WORKSPACE/redisjson.so`)